pkg/task/task.go

```diff
@@ -919,6 +921,17 @@ func (m *taskExecutor) getOEV1alpha1SP() (err error) {
 	return
 }
 
+// getOEV1alpha1SP will get the CstorPool as specified in the RunTask
+func (m *taskExecutor) getOEV1alpha1CSP() (err error) {
```

Comment:
can we have it in new `idiomatic maya` way.
like -
https://github.com/openebs/maya/blob/e01d08c5cac565c7ad216412fe74edaadf94a8a3/pkg/task/task.go#L936
https://github.com/openebs/maya/blob/e01d08c5cac565c7ad216412fe74edaadf94a8a3/pkg/task/task.go#L951

Reply:
This is out of scope of this PR